dev.Dockerfile

```diff
@@ -3,11 +3,12 @@ ENV PYTHONUNBUFFERED=1 POETRY_VERSION=1.7.0
 
 RUN pip3 install poetry==$POETRY_VERSION
 
-WORKDIR /install_temp
-COPY pyproject.toml poetry.* /install_temp/
+WORKDIR /extension
 
-RUN poetry update && poetry install --with dev,devmock,runtime,sdk
+ADD . /extension
 
-ADD ./adobe_vipm /extension
+RUN poetry update && poetry install --with dev,devmock,runtime,sdk && poetry install --only-root
```

Comment:
The last `poetry install --only-root` in theory is not needed because the previous `poetry install --with dev,devmock,runtime,sdk` include the root installation unless you call it with --no-root.

Reply:
Done